libs/features/personalization/personalization.js

```diff
@@ -515,6 +540,7 @@ export async function runPersonalization(info, config) {
     experiment,
     blocks: selectedVariant.useblockcode,
     fragments: selectedVariant.replacefragment,
+
```

Comment:
Potential redundant empty space

Reply:
not sure what you mean here.

Reply:
I think he means "delete this empty line"

Reply:
line removed